script.js

```diff
@@ -0,0 +1,112 @@
+const minNumber = (firstNumber, secondNumber, thirdNumber, fourthNumber) => {
+    if (typeof (firstNumber && secondNumber && thirdNumber && fourthNumber) === "string") {
+      firstNumber = Number(firstNumber);
+      secondNumber = Number(secondNumber);
+      thirdNumber = Number(thirdNumber);
+      fourthNumber = Number(fourthNumber);
+    }
+    if (typeof (firstNumber && secondNumber && thirdNumber && fourthNumber) === "number") {
+      // console.log(Math.min(firstNumber, secondNumber, thirdNumber, fourthNumber));
+      return Math.min(firstNumber, secondNumber, thirdNumber, fourthNumber);
+    } else {
+      // console.log("Wrong number!");
+      return "Something went wrong! There is no minimum number!";
+    }
+  };
+  minNumber("6", 8, null, "-1");
+  
+  const greatestCommonDivider = (x, y) => {
```

Comment:
It will be good to validate zero
<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191680369-2cf7b29b-c1ad-4def-9f1e-182e5ff32c0e.png">